zebra-chain/src/parameters/network/tests/vectors.rs

```diff
@@ -279,13 +286,14 @@ fn check_network_name() {
 fn check_full_activation_list() {
     let network = testnet::Parameters::build()
         .with_activation_heights(ConfiguredActivationHeights {
-            nu5: Some(1),
+            nu6_1: Some(1),
```

Comment:
Please add a comment on why it is ok to do this and when it is going to need to be changed if ever.

Reply:
```suggestion
            // Update this to be the latest network upgrade in Zebra, and update
            // the code below to expect the latest number of network upgrades.
            nu7: Some(1),
```